tasks/utilities/task_rasusa.wdl

```diff
@@ -21,7 +21,7 @@ task rasusa {
     #  --frac [FLOAT] Subsample to a fraction of the reads - e.g., 0.5 samples half the reads
     #  --num [INTEGER] Subsample to a specific number of reads
     String? bases
-    Float coverage 
+    Float coverage = 250
```

Comment:
this is going to impact rasusa standalone workflow where we want the user to specify the coverage. if it's not a required variable for them, then it will be forgotten. I am not in favor of setting this to have a default.

Reply:
okay, i take it back it won't impact those values, but 250 is way lower than the default of 150 for rasusa. this is a weird default to pick